runners/generic_runner.go

```diff
@@ -0,0 +1,18 @@
+package runners
+
+import "sigs.k8s.io/controller-runtime/pkg/manager"
+
+type OperandInitializer struct {
+	cb func()
+}
+
+func NewOperandInitializer(cb func()) manager.Runnable {
```

Comment:
I wonder if the type here should be `func() error` to allow to return the error from the operand initialization.